Leetcode/EigenDemo/eigenPractice.hpp

```diff
@@ -0,0 +1,28 @@
+//
+//  eigenPractice.hpp
+//  EigenDemo
+//
+//  Created by dsh on 2021/7/18.
+//
+
+#ifndef eigenPractice_hpp
+#define eigenPractice_hpp
+
+#include <stdio.h>
+#include <iostream>
+
+#include "Eigen/core"
+#include "Eigen/Dense"
+
+using namespace Eigen;
+
+#define MATRIX_SIZE 50
+
+class eigenPractice {
```

Comment:
```suggestion
class EigenPractice {
```

https://google.github.io/styleguide/cppguide.html#Type_Names

Type names start with a capital letter and have a capital letter for each new word, with no underscores.